src/go/rpk/pkg/cli/cluster/config/get.go

```diff
@@ -69,3 +86,34 @@ output, use the 'edit' and 'export' commands respectively.`,
 
 	return cmd
 }
+
+func getCloudConfig(ctx context.Context, cfg *config.Config, p *config.RpkProfile, configName string) (string, error) {
+	cloudClient := publicapi.NewCloudClientSet(cfg.DevOverrides().PublicAPIURL, p.CurrentAuth().AuthToken)
+	req := connect.NewRequest(&controlplanev1.GetClusterRequest{
+		Id: p.CloudCluster.ClusterID,
+	})
+
+	cluster, err := cloudClient.Cluster.GetCluster(ctx, req)
```

Comment:
There is no endpoint to retrieve configuration properties nor filter for a single property? In order to get a single cluster config property we have to retrieve ALL the cluster properties?

Reply:
No yet, but I know controlplane implements `read_path` for get requests, but I didn't see that attribute in public-api

Reply:
but, what is the problem with get all configs